app/models/competition.rb

```diff
@@ -2661,6 +2664,18 @@ def disconnect_all_payment_integrations
     competition_payment_integrations.destroy_all
   end
 
+  def can_change_registration_system?
+    return false unless registration_not_yet_opened?
```

Comment:
The implicit NOT in `unless` and the (explicit) NOT in the method name cancel each other out. Is there a method like `registration_already_open?`?

Reply:
Created `after_registration_open`. To satisfy this. Naming is a bit tricky, as we're trying to communicate that the registration_open date is in the past, not just whether registration is _currently_ open